slideview/src/main/java/ng/max/slideview/SlideView.java

```diff
@@ -155,13 +155,13 @@ public void setStokeColor(@ColorInt int color) {
         Util.setDrawableStroke(slideBackground, color);
     }
 
-    public void setReverseSlide(boolean reverseSlide) {
-        _setReverseSlide(reverseSlide);
+    public void setReversed(boolean reversed) {
+        _setReversed(reversed);
     }
 
-    private void _setReverseSlide(boolean reverseSlide) {
-        mReverseSlide = reverseSlide;
-        if (reverseSlide) {
+    private void _setReversed(boolean reversed) {
```

Comment:
Do we really need this method? I think we should move the code in this method to the public one `setReversed()` since the only thing the public one does is to call this one.

Reply:
You're right. Also in the init(), there is no problem to call the public method too.